react/src/components/dropdown/SprkDropdown.js

```diff
@@ -127,14 +154,15 @@ class SprkDropdown extends Component {
           additionalClasses={classNames(
             'sprk-c-Dropdown__trigger',
             { 'sprk-u-mrs': variant === 'informational' },
-            additionalTriggerClasses,
+            triggerAdditionalClasses,
           )}
           aria-expanded={isOpen}
           aria-haspopup="listbox"
           aria-label={screenReaderText || triggerText}
           data-analytics={analyticsString || 'undefined'}
           data-id={idString || 'undefined'}
```

Comment:
I know this wasn't part of the AC, but we should update this. If there isn't an `analyticsString` or `data-id` they shouldn't be added.